internal/ingress/controller/kong.go

```diff
@@ -112,11 +113,44 @@ func (n *NGINXController) OnUpdate(ingressCfg *ingress.Configuration) error {
 	return nil
 }
 
+// Duplicated by github.com/kubernetes/ingress-nginx/internal/ingress/controller/controller.go#getServiceClusterEndpoint
```

Comment:
Can you remove this line and put a more meaning comment documenting the method's purpose?